neural_compressor/onnxrt/algorithms/weight_only/algo_entry.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from pathlib import Path
+from typing import Dict, Tuple, Union
+
+import onnx
+
+from neural_compressor.common.logger import Logger
+from neural_compressor.common.utility import RTN_WEIGHT_ONLY_QUANT
+from neural_compressor.onnxrt.quantization.config import RTNWeightQuantConfig
+from neural_compressor.onnxrt.utils.utility import register_algo
+
+logger = Logger().get_logger()
+
+
+###################### RTN Algo Entry ##################################
+@register_algo(name=RTN_WEIGHT_ONLY_QUANT)
+def rtn_quantize_entry(
+    model: Union[Path, str],
+    configs_mapping: Dict[Tuple[str, callable], RTNWeightQuantConfig],

```

Comment:
As we discussed, expand quant_config in algo_entry.py and replace this param name with quant_config